.github/workflows/comments-to-issues.yml

```
@@ -0,0 +1,31 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: TODO to Issue
+        uses: alstr/todo-to-issue-action@v5
+        with:
+          INSERT_ISSUE_URLS: true
+          CLOSE_ISSUES: true
+      
+      - name: Commit and Push Changes
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2006:style:2:7: Use $(...) notation instead of legacy backticks `...`

(shellcheck)